projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -341,9 +341,10 @@ export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelec
         super.open(Object.assign({}, this._overlayDefaults, this.overlaySettings, overlaySettings));
     }
 
-    public inputGroupClick(event: MouseEvent) {
-        if ((event.target as HTMLElement).className !== 'igx-input-group__hint') {
-            this.toggle();
+    public inputGroupClick(event: MouseEvent, overlaySettings?: OverlaySettings) {
+        const targetClass = (event.target as HTMLElement).className;
+        if (!targetClass.includes('igx-input-group__hint')) {
```

Comment:
Really don't like hardcoding a class name from another component/directive.
Do you have a reference to the IgxHintDirective? You can grab it as a child and read the ElementRef so you can compare with the target element.